src/main/resources/openapi/usermanagement.yaml

```diff
@@ -19,12 +19,20 @@ paths:
       parameters:
         - in: query
           name: courthouse
-          required: true
+          required: false
           schema:
             $ref: "#/components/schemas/CourthouseId"
+        - in: header
+          name: Email-Address
```

Comment:
I think this should be lower snake-case, so `email_address`

Reply:
Hyphenated-Pascal-Case appears to be the expectation per HMCTS standards (https://hmcts.github.io/restful-api-standards/#132). Concious we use snake_case elsewhere for headers but I've chosen to align with the standard here.